src/js/models/appUser.js

```diff
@@ -13,7 +13,9 @@ var AppUser = module.exports = Backbone.AssociatedModel.extend({
     },
 
     parse: function(data) {
-        return data.appUser;
+        var appUser = data.appUser;
+        appUser.signedUpAt = new Date(data.appUser.signedUpAt);
```

Comment:
This is necessary because the `signedUpAt` is stored as a number in the backend?